### PR TITLE
Implement popup box for sandbox disable

### LINKED ISSUE #885: Failing Kiwix if started from Windows Network Drive (was: Kiwix does not start on Windows: "QT5Core.dll was not found")
I noticed that Kiwix (version: kiwix-desktop_windows_x64_2.3.0) can't start properly from mapped network shares.
I got "The code execution cannot proceed because QT5Core.dll was not found. Reinstalling the program may fix this problem."
On regular drive everything works fine.

### PR DESCRIPTION
**Pull request by sgourdas**
This fixes #885 by implementing a popup box for windows to ask the user if he wants to disable the sandbox.

### CODE REVIEW

**Comment by sgourdas** [comment]
@kelson42 can we merge this?

File: src/main.cpp [hunk 1]
```diff
@@ -26,7 +26,9 @@ int main(int argc, char *argv[])
     std::string driveLetter = kiwix::getExecutablePath().substr(0, 3);
     UINT driveType = GetDriveTypeA(driveLetter.c_str());
     if(driveType == DRIVE_REMOTE) {
-        int msgboxID = MessageBoxA(NULL, gt("disable-sandbox"), gt("about-kiwix-desktop-title"), MB_YESNO | MB_ICONQUESTION);
+        const char* message = gt("disable-sandbox").toStdString().c_str();
+        const char* title = gt("about-kiwix-desktop-title").toStdString().c_str();
+        int msgboxID = MessageBoxA(NULL, message, title, MB_YESNO | MB_ICONQUESTION);
```
Comment (mgautierfr) [inline src/main.cpp:31]: How does this behave with translation using alphabet outside of ascii ?
We may use the wide version as `MessageBoxExW`.

Comment (sgourdas) [reply src/main.cpp:31]: Good point. i will make that change.

**Review by mgautierfr — CHANGES_REQUESTED** [review]

File: src/main.cpp [hunk 2]
```diff
@@ -26,7 +26,9 @@ int main(int argc, char *argv[])
     std::string driveLetter = kiwix::getExecutablePath().substr(0, 3);
     UINT driveType = GetDriveTypeA(driveLetter.c_str());
     if(driveType == DRIVE_REMOTE) {
-        int msgboxID = MessageBoxA(NULL, gt("disable-sandbox"), gt("about-kiwix-desktop-title"), MB_YESNO | MB_ICONQUESTION);
+        const wchar_t* message = gt("disable-sandbox").toStdWString().c_str();
+        const wchar_t* title = gt("about-kiwix-desktop-title").toStdWString().c_str();
```
[the remark this answers is not in the document]
Comment (sgourdas) [reply src/main.cpp:30]: You are right. Excuse my ignorance.

[the remark this answers is not in the document]
Comment (mgautierfr) [reply src/main.cpp]: You are all excused. No one can know everything. And even if this is a kind of classic bug, it is not a trivial one to catch (that is why it is a classic bug)

**Review by mgautierfr — APPROVED** [review]
LGTM !

**Pull request merged**